HACKING.md

```diff
@@ -312,6 +312,5 @@ You can troubleshoot errors on the [Actions Tab](https://github.com/canonical/op
 
 13. Announce the release on [Discourse](https://discourse.charmhub.io/c/framework/42) and [Matrix](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
 
-14. Open a PR to change [version.py][ops/version.py]'s `version` to the expected
-   next version, with "+dev" appended (for example, if 3.14.1 is the next expected version, use
-   `'3.14.1.dev0'`).
+14. Open a PR to change [version.py](ops/version.py)'s `version` to the expected
+   next version, with ".dev0" appended (for example, if `'3.14.1'` --> `'3.14.1.dev0'`).
```

Comment:
The "if 3.14.1 --> 3.14.1.dev0" reads funny to me. How about, "if 3.14.1, change to 3.14.1.dev0"?

Reply:
I don't understand why we're changing the language at all. It's clearer as it was.

Reply:
reverted the language change, thanks